grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/StandardDao.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.typedb.examples.fraud.db;
+
+import java.util.Set;
+
+public interface StandardDao<T> {
```

Comment:
Is `StandardDao` the best name for this? I feel like just `Dao` is cleaner. If the intent is to be explicit, following the rest of the naming in this example, perhaps `GenericDao`?

Reply:
@SullivanDaly Let's go with Dao